AngouriMath/Core/FiniteSetBuilder.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AngouriMath.Core.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static AngouriMath.Entity;
+using static AngouriMath.Entity.Set;
+
+namespace AngouriMath.Core
+{
+    // TODO: is it useful for the user to have this class?
```

Comment:
Just use a list??

Reply:
Having a separate type might be better, if something needs to be added